src/lib/components/chat/ChatWindow.svelte

```diff
@@ -353,16 +355,33 @@
 				<p>
 					Model:
 					{#if !assistant}
-						<a href="{base}/settings/{currentModel.id}" class="hover:underline"
-							>{currentModel.displayName}</a
-						>{:else}
-						{@const model = models.find((m) => m.id === assistant?.modelId)}
-						<a
-							href="{base}/settings/assistants/{assistant._id}"
-							class="inline-flex items-center border-b hover:text-gray-600 dark:border-gray-700 dark:hover:text-gray-300"
-							>{model?.displayName}<CarbonCaretDown class="text-xxs" /></a
-						>{/if} <span class="max-sm:hidden">·</span><br class="sm:hidden" /> Generated content may
-					be inaccurate or false.
+						{#if models.find((m) => m.id === currentModel.id)}
+							<a
+								href="{base}/settings/{currentModel.id}"
+								class="inline-flex items-center hover:underline"
+								>{currentModel.displayName}<CarbonCaretDown class="text-xxs" /></a
+							>
+						{:else}
+							<span class="inline-flex items-center line-through dark:border-gray-700">
+								{currentModel.id}
```

Comment:
```suggestion
								Model: {currentModel.id}
```

nit: maybe add this label "Model: id" so that it is clear to the user we are talking about model. But up to you